src/commands/bigQueryCostEstimate.ts

```diff
@@ -14,12 +14,14 @@ export class BigQueryCostEstimate {
     private telemetry: TelemetryService,
   ) {}
 
-  async estimateCost() {
+  async estimateCost({ returnResult }: { returnResult?: boolean }) {
```

Comment:
How is this change related?

Reply:
@saravmajestic

Reply:
@mdesmet this PR was supposed to have bigquery cost estimate module as well in the insights panel. That was missing after rebasing with core integration refactor. I have added it again as it is part of insights panel